micro-focus-service-manager/release_notes.md

```diff
@@ -1,5 +1,8 @@
 #### What's Added
 - Added the following new actions and its playbooks
-  - Create Change
-  - Get Change List
-  - Get Change Request
+  
```

Comment:
- Update Change
- Create RF
- Get RF
- Update RF for attachment

Reply:
Also update RF with full form.

Reply:
updated with Rf full form